lib/chef/resource/chef_client_cron.rb

```diff
@@ -213,7 +213,7 @@ def client_command
         #
         def log_command
           if new_resource.append_log_file
-            "-L #{::File.join(new_resource.log_directory, new_resource.log_file_name)}"
+            ">> #{::File.join(new_resource.log_directory, new_resource.log_file_name)} 2>&1"
```

Comment:
I feel like we had a reason for this at one point, but exactly what that was escapes me at the moment.

Reply:
:syslog or :winevt?  there was different behavior in chef-15 around the logging and all the magical wiring that might have wound up with -L being preferred.

it does seem like there needs to be more options available to use -L or not and support syslog and winevt.

Reply:
Without a comment or a test, I'm gonna argue it wasn't that important. ;)

This clearly is "better" in the sense you can catch `STDERR`.

I also agree that options for syslog/winevt would be welcomed by many.

Reply:
I think it was some sort of formatted thing, but this is probably better overall.

Reply:
I would definitely argue that this behavior is correct and anyone broken by it needs to engineer an additional solution